tests/functional_tests/t0_main/langchain/t1_v1.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import sys
+
+import wandb
+
+
+def simple_fake_test():
+    from langchain.chains import LLMChain
+    from langchain.llms.fake import FakeListLLM
+    from langchain.prompts import PromptTemplate
+
+    llm = FakeListLLM(responses=[f"Fake response: {i}" for i in range(100)])
+
+    prompt = PromptTemplate(
+        input_variables=["product"],
+        template="What is a good name for a company that makes {product}?",
+    )
+
+    chain = LLMChain(llm=llm, prompt=prompt)
+
+    for i in range(10):
+        chain(f"q: {i} - {datetime.datetime.now().timestamp()}")
+
+
+def main():
+    if sys.version_info < (3, 8):
```

Comment:
we run functional tests (yea tests) only on py3.7 so you are never testing the `WandbTracer` code currently